test/e2e/adminUI/pages/home.js

```diff
@@ -0,0 +1,95 @@
+module.exports = {
+	elements: {
+		dashboardHeader: '.dashboard-heading'
+	},
+	sections: {
+		accessGroup: {
+			selector: '.dashboard-group:nth-child(1)',
```

Comment:
comment for later:  we should replace positional selectors like this with xpath fixed selectors.